src/containers/reportArticle.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import ReportArticleForm from '../components/articles/reportArticleForm';
+import { reportArticle } from '../actions/articleActions/reportArticleActions';
+
+export class ReportArticle extends React.Component {
+
+	state = {
+		issue: ''
+	};
+
+	handleChange = (event) => {
+		this.setState({
+			issue: event.target.value
+		});
+	};
+
+	handleSubmit = (event) => {
+		event.preventDefault();
+		this.props.dispatch(
+			reportArticle({...this.state}, this.props.data.articlesReducer.article.slug)
+		);
+	};
+
+	handleClick = () => {
+		$('#reportModal').modal('close');
+	};
+
+	render() {
+		return (
+			<ReportArticleForm
+				issue={this.state.issue}
```

Comment:
please destructure the`state` object here

Reply:
It has has been resolved.